src/control/cmd/agent/main.go

```diff
@@ -57,17 +57,69 @@ type cliOptions struct {
 	Insecure   bool       `short:"i" long:"insecure" description:"have agent attempt to connect without certificates"`
 	RuntimeDir string     `short:"s" long:"runtime_dir" description:"Path to agent communications socket"`
 	LogFile    string     `short:"l" long:"logfile" description:"Full path and filename for daos agent log file"`
+	NetworkDbg bool       `short:"n" long:"netdebug" description:"Enable extended network device scan debug output"`
 	Version    versionCmd `command:"version" description:"Print daos_agent version"`
+	NetScan    netScanCmd `command:"scan" description:"Perform local fabric scan"`
```

Comment:
I think this is a good idea... The agent has the ability to do it, so might as well expose it. I suggest "net-scan" to make it more obvious what it's doing when shown in demos/examples, though.

Reply:
+1

Reply:
Done.